tests/test_shared_tools.py

```diff
@@ -232,3 +226,102 @@ def test_version_is_valid():
     assert type(v) is str
     dots = [i for i, c in enumerate(v) if c == '.']
     assert len(dots) == 2
+
+
+@pytest.mark.xfail(raises=IndexError, strict=True)
+def test_limit_inds_error(tmp_path):
+    """IndexError on corner.
+
+    This test throws an error by trying to index cell 1800 of a 30x60 array.
+    This exceeds the limit of the array. I suspect this is a bug with the
+    unravel in shared_tools.
+
+    The xfail should be removed when the bug is fixed.
```

Comment:
Thanks for flagging this test as well as leaving the segfault issues in the below commented out tests. Will be helpful when debugging to see if these get fixed.